hack/local-up-kubeedge.sh

```diff
@@ -46,6 +52,22 @@ function check_prerequisites {
   fi
 }
 
+function verify_crio_installed {
```

Comment:
We can also move verify_crio_installed to hack/lib/install.sh.

Reply:
Resolved
![image](https://github.com/kubeedge/kubeedge/assets/98945276/dba50d00-9de1-4015-84ea-f55a8aed69cf)